apps/console/src/features/core/configs/routes.tsx

```diff
@@ -1680,6 +1680,20 @@ export const getAppViewRoutes = (useExtendedRoutes: boolean = false): RouteInter
                 path: AppConstants.getPaths().get("USERSTORES"),
                 protected: true,
                 showOnSidePanel: true
+            },
+            {
+                category: "extensions:manage.sidePanel.categories.userManagement",
+                component: lazy(() => import("../../provisioning/pages/outbound-provisioning-settings")),
+                exact: true,
+                icon: {
+                    icon: <UserGroupIcon className="icon" fill="black" />
+                },
+                id: "outboundProvisioningSettings",
```

Comment:
Have the CRUD scopes for this route id already been added to deployment.config.json?

Reply:
Fixed 8fdd53d3d07b76a7666f7d305cf20677598271ba